### ISSUE TITLE
Hotkey Ctrl+Alt+K does not hide the selected units info.

### ISSUE THREAD

**Issue by pjmagee** [Ahli/Galaxy-Observer-UI#70]
Hi, I'm not sure where is the best place to report this bug, however I found this is from what you created your shared interface from and so I thought i'll report it here.

I am currently writing an automated spectator in C#. I thought the default interface is not as good as this one and so I am hoping to use the AhliObs 0.66 version. During prototyping my idea I implemented hotkey automation based on events that occur in the game and realised that the Ctrl+Alt+K hotkey does not work. I also see there is a small option icon at the top near the timer, so i clicked that and also found the uncheck option for this hotkey DID work, just that the actual key bind specified does not.

The other default hotkeys seem to be working fine.

Great interface btw.

**Comment by Ahli** [comment]
Thank you for reporting that issue!
This might be the best place to report this. :)

I will check it out when I have time. Unfortunately, I cannot work on it during the week as my laptop broke and I am waiting for the new one to be assembled. I hope I will find some time this weekend.

An automated spectator program sounds fun. I wonder how you interface with the game.

**Comment by pjmagee** [comment]
I have the project on Github, It's prototype at the moment but it's actually quite easy. I parse replay files ahead of time, i analyse 10 seconds into the future every 10 seconds of the current game time, then i can prioritise which hero from any team needs to be focused, and in the order that they die, or potentially even highlight players who get a penta kill for example. 

I use OCR which takes a screenshot of the game every second or so, so that my software is syncronised with the true game time so it knows the moment in time of the replay it needs to go from. I use native Windows calls which send messages to the game (key numbers 1 to 0) are really all that's needed, however I also have some detection which can send Hide Chat and Shift+Z for maximum zoom at the start of the game.

**Comment by Ahli** [comment]
Can you reproduce your issue with my current (unreleased) version, too?
https://www.dropbox.com/s/xxsuo5cvgbt42m5/AhliObs%200.67.StormInterface?dl=1

It works for me in that version. Could some other application eat that key combination, so it never reaches the game?
Can you try editing the settings of AhliObs using my Settings Editor and use another key combination:
https://www.dropbox.com/s/3f2a9ta6sbjzmrs/Observer%20UI%20Settings%20Editor%20Setup.exe?dl=1

The code for that hotkey and the other ones in the menu you found are identical and it works as expected for me. So, I have to assume your environment blocks that key combination (or the game has an incompatibility with your system, etc).

**Comment by pjmagee** [comment]
Hey, thanks for the response, I tried out your new file but it seems that one also didnt work for me. It must be something to do with my system and the alt key combination (I have no idea what that would be). If it's working for you then i guess we know it's not a bug in the game. 

I used your editor and changed the default keybinds to remove the ALT key. It now works with the updated version. I can close the ticket now, thanks!

I will customise the hotkeys as this solves the problem for me.
